File: COVID checker.java
import java.util.Scanner;

public class CovidSymptomChecker {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        String checkAgain;

        do {
            
            System.out.println("Do you have a fever?");
            boolean hasFever = scanner.nextLine().equalsIgnoreCase("Yes");

            System.out.println("Do you have a cough?");
            boolean hasCough = scanner.nextLine().equalsIgnoreCase("Yes");

            System.out.println("Do you have difficulty in breathing?");
            boolean hasDifficultyBreathing = scanner.nextLine().equalsIgnoreCase("Yes");

            
            if (!hasFever && !hasCough && !hasDifficultyBreathing) {
                System.out.println("You do not exhibit significant COVID-19 symptoms. Stay safe!");
            } else {
                System.out.println("You may have COVID-19 symptoms. Please consult a healthcare professional.");
            }

           
            System.out.println("Do you want to check symptoms again? (Yes or No)");
            checkAgain = scanner.nextLine();
            
        } while (checkAgain.equalsIgnoreCase("Yes"));

        scanner.close();
    }
}

File: COVID checker.py
print("Answer:Yes or No")
print("Question")
while True:    
    fever = input("Do you have a fever?").strip().lower()
    cough = input("Do you have cough?").strip().lower()
    breathing = input("Do you have difficulty in breathing?").strip().lower()        
    if (fever == "no" and cough == "no" and breathing == "no"):        
        print("You do not exhibit significant COVID-19 symptoms. Stay safe!")
    else:
        print("You may have COVID-19 symptoms. Please consult a healthcare professional.")   
    repeat = input("Do you want to check symptoms again? Yes or no: ").strip().lower()
    if (repeat != 'Yes'):
        print("Thank you for using the symptom checker. Stay safe!")
        break
      